tests/test_sensors.py

```diff
@@ -34,8 +33,28 @@ def test_imu_sensor(show_viewer):
         ),
     )
 
-    imu_biased = scene.add_sensor(IMUOptions(entity_idx=box.idx, accelerometer_bias=BIAS, gyroscope_bias=BIAS))
-    imu_delayed = scene.add_sensor(IMUOptions(entity_idx=box.idx, read_delay=DT * 2))
+    imu_biased = scene.add_sensor(gs.sensors.IMU(entity_idx=box.idx, acc_bias=BIAS, gyro_bias=BIAS))
```

Comment:
By convention, always using multi lines for options arguments no matter if it could fit on one line. It makes everything more readable.